lua/nvim-tree/fs.lua

```diff
@@ -317,4 +317,27 @@ function M.print_clipboard()
   return api.nvim_out_write(table.concat(content, '\n')..'\n')
 end
 
+local function Copied(content)
```

Comment:
i did not mean the function, the name before was ok, but the string.format should say `Copied %s to system clipboard` instead of Copy

Reply:
Sorry, I misunderstood.